.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,8 @@ assignees: ''
 
 ---
 
+***PLEASE FILL THE FOLLOWINGS AS MUCH AS POSSIBLE FOR FASTER SOLUTION***
+
 **Describe the bug**
 A clear and concise description of what the bug is.
```

Comment:
I think we should add the stackoverflow and google group links to every template including bug report and feature request

Reply:
added